student.rb

```diff
@@ -1,15 +1,40 @@
+require_relative 'classroom'
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :age, :parent_permission, :classroom
+
   def initialize(classroom, name = 'unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
+    @name = name
+    @age = age
+    classroom.add_student(self)
   end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍